### ISSUE TITLE
你好请问使用operator方式部署支持使用外置的etcd吗？文档怎么看到

### ISSUE THREAD

**Issue by 370569218** [karmada-io/karmada#4662]
你好请问使用operator方式部署支持使用外置的etcd吗？文档怎么看到配置方式

**Comment by XiShanYongYe-Chang** [comment]
Hi @370569218, do you mean that the operator function meets the requirements but the document does not contain the content?
Or do you expect this feature to be added in operator?

Ask the module owner to help take a look~
/cc @calvin0327

**Comment by calvin0327** [comment]
indeed, operator is not support the way to connect to a external etcd.

**Comment by calvin0327** [comment]
@370569218  now, we have  corresponding api for external etcd in `karmada` struct. are you interested in the rest of work?

**Comment by XiShanYongYe-Chang** [comment]
Kindly ping @370569218
